cmake/Install.cmake

```diff
@@ -18,8 +18,6 @@ else()
   set(CPACK_PACKAGE_FILE_NAME "${CMAKE_PROJECT_NAME}-${CPACK_PACKAGE_VERSION}-${CMAKE_SYSTEM_NAME}-${TARGET_ARCH}-${CMAKE_BUILD_TYPE}")
 endif()
 
-set(CPACK_PACKAGING_INSTALL_PREFIX "/${CMAKE_PROJECT_NAME}-${CPACK_PACKAGE_VERSION_MAJOR}-${CPACK_PACKAGE_VERSION_MINOR}-${CPACK_PACKAGE_VERSION_PATCH}")
```

Comment:
@jmarrec I wonder if this is the culprit for the change in Ubuntu installer structure.  If it is, and we want it back the original way, I think we can simply re-add this line, but it would be good to put it in an `IF ( UNIX AND NOT APPLE )` block with a comment on why it is needed.

If it's not this line, then I'm not really sure what it would have been that changed in this release.  I'd have to `git bisect` it to find when it changes.

Reply:
Entirely possible. Without this change, the file structure that got created on Mac was not compatible with QtIFW. I didn't "see" any ramifications on the other platforms when I did this, but I wasn't looking too closely. =( Sorry.

Reply:
Pretty sure it's this line indeed.

I noticed the change isn't just on Ubuntu but on all platforms, so it'd make more sense to change to the OpenStudio FindEnergyPlus.cmake file


Just a side note, this change in structure may get pretty confusing for some (very? limited numbers of) users: if you just try to extract the `tar.gz` to `/usr/local` with `tar -xzf EnergyPlus-9.1.0-d830726d9e-Darwin-x86_64.tar.gz --strip 1 -C ./usr/local/`, it'll no longer end up in `/usr/local/EnergyPlus-9-1-0` but directly in `/usr/local/`

Reply:
I modified OpenStudio's Cmake stuff to match the new structure, cf https://github.com/NREL/OpenStudio/pull/3427